src/frontend/src/pages/myfeeds.js

```diff
@@ -14,6 +14,28 @@ const useStyles = makeStyles(theme => ({
 
 export default function MyFeeds() {
   const classes = useStyles();
+  const [feedAuthor, setFeedAuthor] = useState('');
+  const [feedUrl, setFeedUrl] = useState('');
```

Comment:
Could these two lines be refactored into one? i.e.
```js
const [feedAuthor, feedUrl, setFeedUrl, setFeedAuthor] = useState('');
```

Reply:
Thanks for the quick feedBack, I'm not too sure since we're using React Hooks. Took a quick look and this is what I came across 

![image](https://user-images.githubusercontent.com/18711727/75402702-8b803380-58d3-11ea-80d3-1ec2d4d400e8.png)

Reply:
Ah, I see. Probably not, then.!